### DISCUSSION TITLE
Laravel 10 Support

### DISCUSSION THREAD

**Discussion by SergkeiM** [oddvalue/laravel-drafts#14] [Ideas]
Now that Laravel 10 is out, any plans to add support for it?

**Comment by oddvalue** [comment]
Hey, yeah I do plan to. I did have a branch that worked with v10 but the test suite didn't like it and I didn't have time to dig into it. I will pick it up again soon, though.

**Comment by oddvalue** [comment]
@Froxz v1.2.0 now has support for Laravel 10 :slightly_smiling_face:

**Comment by SergkeiM** [comment]
Hi @oddvalue Awesome, Thank you.
